pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -462,7 +462,7 @@ const forwardTemplate = `{{- define "forward" }}
 	{{ if .Target.Secret }}
 	<security>
 		self_hostname "#{ENV['NODE_NAME']}"
-		shared_key {{ .Name }}
+		shared_key {{ .SecretValue "shared_key" }}
```

Comment:
This means the cleartext shared_key will be in the fluentd configmap, as opposed to doing something like https://github.com/openshift/origin-aggregated-logging/blob/release-3.11/fluentd/configs.d/input-post-forward-mux.conf#L9
```
  shared_key    "#{File.open('/etc/fluent/muxkeys/shared_key') do |f| f.readline end.rstrip}"
```
so that the key is either in the mounted secret, or in memory
@alanconway thoughts?

Reply:
Relying on fluent to load the key would actually simply the code and this change I believe as there would be no need to actually load and unencode the secret.